core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -165,7 +115,7 @@ class AnalyzerManager(
       for {
         (module, analyzer) <- analyzers
       } analyzer forward ReloadExistingFilesEvent
-      if(analyzers.isEmpty)
+      if (analyzers.isEmpty)
```

Comment:
I'd have done it as a `if (analyzers.isEmpty) ... else for ...` to avoid reasoning about the implied no-op in the `for` comprehension.